qiskit/tools/wigner_function_tomography.py

```diff
@@ -0,0 +1,114 @@
+"""
+
+Wigner Function Tomography Module
```

Comment:
I think these functions should be moved into the tomography.py module rather than being in their own module